Mu2eG4/src/ConstructMaterials.cc

```diff
@@ -1669,7 +1669,60 @@ namespace mu2e {
       G4Material* ClosePackedExtMonSteelShot = new G4Material(mat.name, density*CLHEP::g/CLHEP::cm3, 1);
       ClosePackedExtMonSteelShot -> AddMaterial(findMaterialOrThrow("MildSteel"), 100*CLHEP::perCent);
     }
+/*
+    mat uniqueMaterialOrThrow("R-449A");
```

Comment:
Is the commented out material being retained as an option?    Generally speaking I would create the material so that you can try options without needed to recompile. Unless there is a good, forward looking reason, I don't like to keep commented out code.